code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -288,73 +287,92 @@
 			replace_beaker(usr)
 			. = TRUE
 		if("dispense_recipe")
-			if(!is_operational || QDELETED(cell))
```

Comment:
Why did you remove this?

Reply:
I don't know why but QDELETED(CELL) caused it to runtime. It still has the !is_oeprational.

Reply:
Show the runtime, this shouldn't be possible since it runs a check on whether or not the cell actually exists at all.